docs/kr.tree

```diff
@@ -305,7 +305,14 @@
 
     <toc-element toc-title="Tools">
         <toc-element toc-title="Build tools">
-            <toc-element id="gradle.md" accepts-web-file-names="using-gradle.html"/>
+            <toc-element toc-title="Gradle">
+                <toc-element id="gradle.md" accepts-web-file-names="using-gradle.html" toc-title="Overview"/>
+                <toc-element id="gradle-backend-tutorial.md" toc-title="Get started with Gradle tutorial"/>
+                <toc-element id="gradle-configure.md" toc-title="Configure your own project"/>
+                <toc-element id="gradle-compiler-options.md" toc-title="Compiler options"/>
```

Comment:
Names of sections are not consistent between each other but is it a crime this time?:) I suppose we can use such names.